chart/values.yaml

```diff
@@ -830,6 +830,14 @@ nats:
         size: "5Mi"
       fileStorage:
         enabled: false
+  # Affinity for pod assignment
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              app.kubernetes.io/name: nats
```

Comment:
should this have release name as well?

Reply:
This all of the labels the sts Pod carries:
```yaml
    app: nats
    app.kubernetes.io/instance: mayastor
    app.kubernetes.io/name: nats
    apps.kubernetes.io/pod-index: "0"
    controller-revision-hash: mayastor-nats-867775cc4b
    openebs.io/logging: "true"
    statefulset.kubernetes.io/pod-name: mayastor-nats-0
```

Additionally, the release name is dynamically input, we can't make nats `tpl` `{{ .Release.Name }}` from the values.

Reply:
oh, indeed we can't :(